pkg/driver/nydus/nydus.go

```diff
@@ -285,14 +307,13 @@ func (d *Driver) convert(ctx context.Context, provider accelcontent.Provider, so
 	convertHooks := converter.ConvertHooks{
 		PostConvertHook: convertHookFunc,
 	}
-	indexConvertFunc := converter.IndexConvertFuncWithHook(
+	indexConvertFunc := IndexConvertFuncWithHook(
```

Comment:
We should consider extending the content store interface to implement pulling-free instead of porting the converter codes, right?